src/date-picker/__tests__/date-picker-calendar.test.tsx

```diff
@@ -273,26 +275,29 @@ describe('Date picker calendar', () => {
         expect(findCalendarHeaderText(wrapper)).toBe('April 2018');
       });
 
-      test('should allow initially selected date to be re-selected with enter', () => {
-        wrapper.findCalendar()!.findSelectedDate().keydown(KeyCode.enter);
+      test.each(selectionKeys)('should allow initially selected date to be re-selected with %s', key => {
```

Comment:
As the keyboard navigation logic was moved to a new file, Codecov revealed that using space key was not covered, only enter.